### DISCUSSION TITLE
Significance of the number in the class attribute

### DISCUSSION THREAD

**Discussion by PlasmaHH** [biati-digital/glightbox#531] [Q&A]
The Examples of the README.md show not only using the class glightbox but also followed by a number, glightbox3 etc. now I first thought that it was for the mentioned use cases of veideo, gallery etc. but upon trying things out these classes seem to have no functionality at all. The readme also doesnt tell anything abou it. So what do they mean?

**Comment by gingerchew** [comment]
It's nothing built into the library, just a quick way to make multiple galleries.

```js
const videos = GLightbox({ selector: '.glightbox3' });
const images = GLightbox({ selector: '.glightbox1923871049823' });
const mySuperSpecialLightBoxThatIsSuperSpecific GLightbox({ selector: '.theres-no-way-this-selector-exists-anywhere-other-than-one-place' });
```
